core/src/main/java/com/linecorp/armeria/common/logging/ResponseLogLevelMapper.java

```diff
@@ -66,6 +66,26 @@ static ResponseLogLevelMapper of(HttpStatusClass statusClass, LogLevel logLevel)
         return log -> log.responseStatus().codeClass() == statusClass ? logLevel : null;
     }
 
+    /**
+     * Creates a new {@link ResponseLogLevelMapper} which returns the specified {@link LogLevel} if the given
+     * class of {@link RequestLog#responseCause()} is equal to the specified {@link Throwable} class.
+     */
+    static ResponseLogLevelMapper of(Class<? extends Throwable> clazz, LogLevel logLevel) {
+        requireNonNull(clazz, "clazz");
+        requireNonNull(logLevel, "logLevel");
+        return log -> {
+            final Throwable t = log.responseCause();
+            if (t == null) {
+                return null;
+            }
+            final Class<? extends Throwable> throwableClass = t.getClass();
+            if (throwableClass == clazz) {
```

Comment:
Should we use `isAssignableFrom()` so that all subclasses of `clazz` is handled?

Reply:
@ikhoon
I addressed this.
Please review again.